doc/thingpedia-tutorial-cat.md

```diff
@@ -89,7 +89,7 @@ dataset @com.thecatapi {
     snippets together, the system knows what argument can be passed to the parameter.
   */
   query (p_count :Number)  := @com.thecatapi.get(count=p_count)
-  #_[utterances=["${p_count:const} cat pictures"]];
+  #_[utterances=["${p_count} cat pictures"]];
```

Comment:
Why this change? Pretty much all count parameters need `:const`.

Reply:
What's the meaning of `const`? I want to introduce it in the nl support section, but not here.

Reply:
oh, I see you explanation above...